docs/hw/display/nv3/vga-stack.rst

```diff
@@ -29,7 +29,7 @@ On NV41:G80, the registers are located in PBUS area:
 - 001388 CONFIG
 - 00138c SP
 
-They are also aliased in the VGA CRTC register space:
```

Comment:
here it should definitely be CRTC - it's called that in all VGA documentation on the planet.